Sources/PointFree/Admin/SendNewBlogPostMailer.swift

```diff
@@ -60,23 +61,13 @@ private let newBlogPostEmailRowView = View<BlogPost> { post in
     ])
 }
 
-public struct NewBlogPostFormData: Codable, Equatable {
-  let nonsubscriberAnnouncement: String
-  let nonsubscriberDeliver: Bool?
-  let subscriberAnnouncement: String
-  let subscriberDeliver: Bool?
-
-  fileprivate enum CodingKeys: String, CodingKey {
-    case nonsubscriberAnnouncement = "nonsubscriber_announcement"
-    case nonsubscriberDeliver = "nonsubscriber_deliver"
-    case subscriberAnnouncement = "subscriber_announcement"
-    case subscriberDeliver = "subscriber_deliver"
-  }
-}
-
 let sendNewBlogPostEmailMiddleware:
-  Middleware<StatusLineOpen, ResponseEnded, Tuple4<User?, BlogPost, NewBlogPostFormData?, Bool?>, Data> =
+  Middleware<StatusLineOpen, ResponseEnded, Tuple4<User?, BlogPost.Id, NewBlogPostFormData?, Bool?>, Data> =
```

Comment:
When routes were in the main `PointFree` module I could make the route fetch blog posts directly from `Current.allBlogPosts`. We can no longer do that in a module, so we fetch id's, and then I have an extra middleware transformer down below to fetch the blog post.